support/cas-server-support-jdbc/src/main/java/org/apereo/cas/adaptors/jdbc/config/CasJdbcAuthenticationConfiguration.java

```diff
@@ -53,18 +49,8 @@
 @ConditionalOnFeatureEnabled(feature = CasFeatureModule.FeatureCatalog.Authentication, module = "jdbc")
 @AutoConfiguration
 public class CasJdbcAuthenticationConfiguration {
-    private static AuthenticationHandler searchModeSearchDatabaseAuthenticationHandler(
-        final SearchJdbcAuthenticationProperties b,
-        final PasswordPolicyContext config,
-        final ConfigurableApplicationContext applicationContext,
-        final PrincipalFactory jdbcPrincipalFactory,
-        final ServicesManager servicesManager) {
-        val h = new SearchModeSearchDatabaseAuthenticationHandler(b, servicesManager, jdbcPrincipalFactory, JpaBeans.newDataSource(b));
-        configureJdbcAuthenticationHandler(h, config, b, applicationContext);
-        return h;
-    }
 
-    private static void configureJdbcAuthenticationHandler(final AbstractJdbcUsernamePasswordAuthenticationHandler handler,
+    public static void configureJdbcAuthenticationHandler(final AbstractJdbcUsernamePasswordAuthenticationHandler handler,
```

Comment:
Can you elaborate on why this is public please? 

This should never be public or protected. Extending a configuration class is anti pattern and bad practice. If you need access to this function, it would be best to move it outside of here.

Reply:
For example, I may want to instantiate my own list of `CustomQueryAuthnHandler` and I need this method to properly build it? Where would you see this utility method? Thx

Reply:
Yup, I figured as much and to build your own, you would either need to extend this class or do something slightly more unpleasant. I don't see a good home for the utility method just yet; maybe create your own JdbcAuthenticationUtils class for now? That should do. Thank you!

Reply:
I created the `JdbcAuthenticationUtils` class. Can I backport this change to 6.6?

Reply:
Backported to: https://github.com/apereo/cas/pull/5554